systemvm/debian/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -476,9 +476,10 @@ def fw_vpcrouter(self):
             self.fw.append(["", "front", "-A NETWORK_STATS_%s -o %s -s %s" %
                             ("eth1", "eth1", guestNetworkCidr)])
 
-            self.fw.append(["nat", "front",
-                            "-A POSTROUTING -s %s -o %s -j SNAT --to-source %s" %
-                            (guestNetworkCidr, self.dev, self.address['public_ip'])])
+            if self.address["source_nat"]:
+                self.fw.append(["nat", "front",
```

Comment:
@Slair1 please fix indent:
```
../debian/opt/cloud/bin/cs/CsAddress.py:481:29: E128 continuation line under-indented for visual indent
../debian/opt/cloud/bin/cs/CsAddress.py:482:29: E128 continuation line under-indented for visual indent
+'[' 1 -gt 0 ']'
+echo 'Pylint failed, please check your code'
Pylint failed, please check your code
```

Reply:
@rhtyd Increased indent and squashed commits into 1

Reply:
@DaanHoogland It looks like the test "test_03_vpc_privategw_restart_vpc_cleanup" is failing on several PRs.  Looking at the logs it seems to be a NullPointerException:

```
java.lang.NullPointerException
	at com.cloud.network.router.NicProfileHelperImpl.createPrivateNicProfileForGateway(NicProfileHelperImpl.java:95)
```

### NicProfileHelperImpl.java (Lines 93 - 95)
```
            final PrivateIpAddress ip =
                    new PrivateIpAddress(ipVO, privateNetwork.getBroadcastUri().toString(), privateNetwork.getGateway(), netmask,
                            NetUtils.long2Mac(NetUtils.createSequenceBasedMacAddress(ipVO.getMacAddress(), NetworkModel.MACIdentifier.value())));
```